vcn-web/src/components/molecules/m-send.jsx

```diff
@@ -0,0 +1,20 @@
+import { h } from 'preact';
+import AInput from '../atoms/a-input';
+import ASvg from '../atoms/a-svg';
+
+const MSend = ({ className = "", ...props}) => {
+
+  return (
+    <AInput type='textarea' name='send' {...props} className={`m-send ${className}`}>
```

Comment:
@jrcarretas what are your thoughts on this the Atom/Molecule/Organism system and prefixes? Is it helpful?

Reply:
I like the idea for the prefixes. I normally open files by searching for filenames. The prefixes would help lessen cognitive load.

The Atom/Molecule/Organism folder management is good too. I would probably organize related components into folders, if possible.